### PR TITLE
added a blog - when grafana meet wechaty

### PR DESCRIPTION
**Pull request by techxel**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [x] Add Blog Header, including title, author, date.
e.g.

``` md
---
 title: "'Score Your Face Photo' a ML&Wechaty practice"
 date: 2017-09-18 09:00 +0800
 author: huyingxi
 ---
 > Author: [@huyingxi](https://github.com/huyingxi/wechaty_selfie) enjoying ML&Wechaty at BUPT
```

- [x] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [x] Embed the photo & video before publishing, save all external file to the blog `/download/2018` directory.
- [x] Add `<!--more-->` section to just show the abstract in blog homepage.
- [x] Add teaser image for the blog
- [x] Add your self as an author to the author data file at <https://github.com/wechaty/wechaty.github.io/blob/master/_data/authors.yml> with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!

[See more](https://github.com/wechaty/wechaty.github.io#contribute-guideline)

### CODE REVIEW

**Comment by lijiarui** [comment]
Since we change the template or wechaty blog, we have a break change:

## Before
You should add your profile at `author.yml`

## After

remove `author.yml`

### 1. Create a file at `_developers` named as `yourname.md` 

 `yourname` should be the same as the author name you put in the blog:
```
---
title: XX
date: XX
author: yourname
categories: XX
tags:
  - XX
image: XX
---
```
### 2. The file should begin with:
```
---
name: Jiarui LI (李佳芮)
site: https://lijiarui.github.io
bio: 句子互动创始人 & CEO，微软人工智能最具价值专家 (AI MVP)
avatar: /assets/developers/lijiarui/avatar.png
email: rui@juzi.bot
twitter: https://weibo.com/u/2175505900
---
```

### 3. Example:

- lijiarui.md: https://github.com/wechaty/wechaty.github.io/blob/master/_developers/lijiarui.md
- It will shown like this on wechaty blog: https://wechaty.github.io/developers/lijiarui/

Looking forward to your profile file at the `_developers` folder!

**Comment by lijiarui** [comment]
Please resolve the conflict and change as I request

File: _data/authors.yml [hunk 1]
```diff
@@ -971,3 +971,18 @@ csjuXYZ:
     - label: "Email"
       icon: "fas fa-fw fa-envelope-square"
       url: "hoc0415@foxmail.com"
+
```
Comment (lijiarui) [inline _data/authors.yml]: Since we have the profile markdown file, so you should remove this introduction commit from authors.yml.

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: _developers/magina.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+name: Maigna Yin (殷临风)
+site: https://github.com/techxel
+bio: focus on Devops & IoT
+avatar: /assets/developers/magina/avatar.jpg
+email: magina.yin@gmail.com
+twitter: 
+---
```
Comment (lijiarui) [inline _developers/magina.md:8]: It could be better for you to add more information in your profile, it can make other developers know you better!

**Review by lijiarui — CHANGES_REQUESTED** [review]

**Comment by lijiarui** [comment]
Please sign the CLA firs then I can look into this PR.

**Comment by techxel** [comment]
image

I have already signed, but I didn't know whether something wrong about it?

**Comment by techxel** [comment]
I think maybe this user which belongs yours also need sign

image

**Comment by lijiarui** [comment]
You should check your username and your email. These two should be unified.

**Comment by techxel** [comment]
I have signed before, at that time everything seems well, when TbhT came to merge, the sign started to be pending.

image

**Comment by techxel** [comment]
It's obvious the user TbhT didn't sign

**Comment by lijiarui** [comment]
If `TbhT` should sign, it means this commit is related to `TbhT`, but your PR should not related to other developer's commit.

Or you can create a new PR.

The precondition of our review is that everyone related to this PR has signed the CLA.

**Comment by huan** [comment]
@TbhT Please sign the SLA under Wechaty org because you have synced the master commits to the branch from @techxel, which blocked this PR to pass the CLA check.

Thank you very much.

**Comment by TbhT** [comment]
> @TbhT Please sign the SLA under Wechaty org because you have synced the master commits to the branch from @techxel, which blocked this PR to pass the CLA check.
> 
> Thank you very much.

DONE.

**Comment by lijiarui** [comment]
As I requested before, please change as I request.

```
It could be better for you to add more information to your profile, it can make other developers know you better!
```

**Comment by techxel** [comment]
> As I requested before, please change as I request.
> 
> ```
> It could be better for you to add more information to your profile, it can make other developers know you better!
> ```

I have already added before, I also find most of the authors weren't so detail as for me. I don't want to publish so much information on the Internet, just want to make some contributions to the project, don't want others to know me. you have rejected me several times, just for the personal information. I think it's not crucial for this project to force people doing something like this

**Comment by huan** [comment]
Hi @techxel !

I'm sorry to see that you canceled your PR due to the review process. You have almost finished all the steps: CLA is signed, CI is green, everything will be ok.

Our previous rejection is not because of your personal information, it's mainly because of the CLA and the CI staff, your developer profile is totally controlled by yourself, and please feel free to add more information to it or not.

So please feel free to create another PR or reopen this PR to submit your great post again, you will be always welcome to come back.

Have a great weekend!

**Comment by techxel** [comment]
@huan  thanks, so good guy you are, and also fairly professional. I appreciated finding this project, and also want to participate in the python and go versions' development, but for some reason and I have my job. I hope this project will be better and better. I have persuaded my friend to buy your token, it's rewarding to pay for such a wonderful product. So I think this way is the best support for you and your partners.

**Pull request closed without merging**